### DISCUSSION TITLE
Make a copy of a graphic view

### DISCUSSION THREAD

**Discussion by nvlong248** [SCADA-LTS/Scada-LTS#2821] [General]
Is it possible to make a copy of a graphic view to another one?

**Comment by fabiodurao** [comment]
This is a much desired feature, but it doesn't exist yet. It would be ideal if we had something similar to what we have for copying datasources and perhaps even a graphical representation management screen, similar to what exists in datasources.

What we have today is the possibility of going to the import and export area, checking the graphic views box, exporting the json, isolating the json only from the screen of interest, copying it to a notepad or even Excel, editing the name of the new screen, the xid and put it inside the json on the Scada-LTS screen and click on import, it takes more work than the previous idea, but it works too.

**Reply by nvlong248** [reply]
thank you. That is a good idea at this time. By the way, do you know how to make copies of several datapoints, together with their event detectors?

**Reply by Limraj** [reply]
Hi @nvlong248,
You can copy the entire data source with point(s):
![image](https://github.com/SCADA-LTS/Scada-LTS/assets/35842300/5fdeb008-1198-41b2-994e-73d7311ea5b2)

**Reply by Limraj** [reply]
Hi @nvlong248, 
from version 2.7.8 it is possible to copy points:
issue: https://github.com/SCADA-LTS/Scada-LTS/issues/2773

We will soon release version 2.7.8

Regards,
Kamil Jarmusik
